### ISSUE TITLE
[1pt] Update the requirements and turn on the kebechet 

### ISSUE THREAD

**Issue by harshad16** [thoth-station/zuul-test-infra#58]
**Is your feature request related to a problem? Please describe.**
Many of the components are on old version of the packages, for exaple: mypy, pre-commit etc.
Lets update them and iterate over the release.

**Describe the solution you'd like**
Manully update of now and turn on kebechet.

**Comment by harshad16** [comment]
Kebechet update is failing in this repo due to below issue:
```
2022-07-06 19:15:46,855   1 ERROR    kebechet:214: An error occurred during run of manager {'configuration': {}} <class 'kebechet.managers.update.update.UpdateManager'> for 'thoth-station/thoth-ops-infra', skipping
Traceback (most recent call last):
  File "/home/user/kebechet/kebechet_runners.py", line 212, in run
    instance.run(**manager_configuration)
TypeError: run() missing 1 required positional argument: 'labels'
2022-07-06 19:15:52,115   1 INFO     kebechet:196: Running manag
```
Tried with issue: #60

**Comment by codificat** [comment]
It seems to me that this was fixed by #59 - it was actually referenced in the description there that it fixes this issue, not sure why it was not auto-closed... maybe you have to specify each `Fixes: XX` in its own separate line?

/close

Please reopen if I read this wrong

**Comment by codificat** [comment]
Oh, I now think I spoke too soon, this seems not completely fixed yet: Kebechet is complaining that there is no `Pipfile` or `requirements.txt` to work with, see #63 

However, not reopening this now as it would duplicate #63 - let's use that to track that problem.
